libpod/container.go

```diff
@@ -455,7 +459,7 @@ func (c *Container) Dependencies() []string {
 		dependsCtrs[c.config.NetNsCtr] = true
 	}
 	if c.config.PIDNsCtr != "" {
-		dependsCtrs[c.config.NetNsCtr] = true
+		dependsCtrs[c.config.PIDNsCtr] = true
```

Comment:
Oops - this needs to be fixed quickly. Can you spin up a separate PR with just this fix?